BLSQ/iaso
# Polio-1328: stock management details table(mock data)

Explain what problem this PR is resolving

Related JIRA tickets :  POLIO-1328

## Self proofreading checklist


## Changes

- Create `PaperTableRow` from `BeneficiariesInfos`
- Create details page:
    - use `ParamsPrefix` prop to manage 2 tables in same page
    - use `WidegtPaper` to show summary details
- add mock data

Note about the tables: I tried using 1 table for both tabs, using prefixed params and the tab value to switch between data, but it was messing up the table's internal state and generating weird pagination bugs

## How to test

Go to polio>vaccine module > stock management 
click the details iconbutton on any row. They all show the same data, but you will see the `id` change in the url
Play around with the tables. You should see the parameters update accordingly in the url
You can see the querystrings for each mocked api call in the console


## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/dbe5e8b1-e1a1-46a2-821a-2ff9810a2412


### Update: changed the layout a bit:
![Uploading Screenshot 2023-12-07 at 15.38.29.png…]()


## Notes

Has to be merged after POLIO-1149
